src/js/step.js

```diff
@@ -159,13 +155,10 @@ export class Step extends Evented {
    * Triggers `destroy` event
    */
   destroy() {
-    if (this.tooltip) {
-      this.tooltip.destroy();
-      this.tooltip = null;
-    }
+    destroyTooltip(this);
```

Comment:
moved this bit in it's own function to match what's happening with `setupTooltip`.